index.js

```diff
@@ -4,27 +4,51 @@ const arrify = require('arrify');
 const isDirectory = require('is-directory');
 const pify = require('pify');
 
-const getGlob = (fp, ext) => path.join(fp, '**', ext || '');
-const getExt = ext => Array.isArray(ext) ? `*.{${ext.join(',')}}` : `*.${ext}`;
-const getPath = fp => fp[0] === '!' ? fp.slice(1) : fp;
+const isDirectoryP = pify(isDirectory);
```

Comment:
Yes, it does. You aren't handling `ENOENT` anywhere. Tests are failing when using it:

```
convert directories to glob - async

  Rejected promise returned by test. Reason:

  Error {
    code: 'ENOENT',
    errno: -2,
    path: 'foo/**',
    syscall: 'stat',
    message: 'ENOENT: no such file or directory, stat \'foo/**\'',
  }
```